<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Yonetici</class>
 <widget class="QMainWindow" name="Yonetici">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grp_kitap">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>251</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Kitaplar</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>21</y>
       <width>251</width>
       <height>531</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rdr_k1">
        <property name="text">
         <string>Kitap1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rdr_k2">
        <property name="text">
         <string>Kitap2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="rdr_k3">
        <property name="text">
         <string>Kitap3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="rdr_k4">
        <property name="text">
         <string>Kitap4</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="rdr_k5">
        <property name="text">
         <string>Kitap5</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="rdr_k6">
        <property name="text">
         <string>Kitap6</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QRadioButton" name="rdr_k7">
        <property name="text">
         <string>Kitap7</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QRadioButton" name="rdr_k8">
        <property name="text">
         <string>Kitap8</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QRadioButton" name="rdr_k9">
        <property name="text">
         <string>Kitap9</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QRadioButton" name="rdr_k10">
        <property name="text">
         <string>Kitap10</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QRadioButton" name="rdr_k11">
        <property name="text">
         <string>Kitap11</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QRadioButton" name="rdr_k12">
        <property name="text">
         <string>Kitap12</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QRadioButton" name="rdr_k13">
        <property name="text">
         <string>Kitap13</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QRadioButton" name="rdr_k14">
        <property name="text">
         <string>Kitap14</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QRadioButton" name="rdr_k15">
        <property name="text">
         <string>Kitap15</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_ycks">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>490</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Çıkış</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>371</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>Ögrenciler</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>371</width>
       <height>521</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QGroupBox" name="grp_ahmet">
        <property name="title">
         <string>Ahmet 192106035</string>
        </property>
        <widget class="QLabel" name="lbl_a1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>141</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_a2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>10</y>
           <width>201</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="grp_bekirhan">
        <property name="title">
         <string>Bekirhan 192106010</string>
        </property>
        <widget class="QLabel" name="lbl_b1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>141</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_b2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>20</y>
           <width>201</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="grp_furkan">
        <property name="title">
         <string>Furkan 192106011</string>
        </property>
        <widget class="QLabel" name="lbl_f1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>131</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_f2">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>10</y>
           <width>211</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="grp_mustafa">
        <property name="title">
         <string>Mustafa 192106012</string>
        </property>
        <widget class="QLabel" name="lbl_m1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>141</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_m2">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>10</y>
           <width>211</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QGroupBox" name="grp_dogukan">
        <property name="title">
         <string>Doğukan 192106013</string>
        </property>
        <widget class="QLabel" name="lbl_d1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>141</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_d2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>10</y>
           <width>201</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QGroupBox" name="grp_mucahit">
        <property name="title">
         <string>Mücahit 192106014</string>
        </property>
        <widget class="QLabel" name="lbl_mu1">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>141</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_mu2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>10</y>
           <width>201</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3"/>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>45</y>
      <width>171</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QComboBox" name="cmb_ogrenciler">
       <item>
        <property name="text">
         <string>Ahmet 192106035</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Furkan 192106011</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mustafa 192106012</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Doğukan 192106013</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mücahit 192106014</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bekirhan 192106010</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDateEdit" name="dte_tarih">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2020</year>
         <month>12</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QDateEdit" name="dte_tarih2">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2020</year>
         <month>12</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btn_ktpekle">
       <property name="text">
        <string>Kitap ekle</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="btn_ktpal">
       <property name="text">
        <string>Kitap al</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
